pipeline/accessibility-insights-web.yml

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The file contents look fine, but I wonder if we can give this file a more descriptive name. Maybe `build-shared.yaml` or `build-common.yaml` ?

Reply:
Changed the file name to build-shared.yaml